packages/ifixit-api-client/index.tsx

```diff
@@ -44,12 +45,14 @@ export class IFixitAPIClient {
    }
 
    async fetch(endpoint: string, init?: RequestInit) {
-      const response = await fetch(
-         `${this.origin}/api/${this.version}/${endpoint}`,
-         {
-            credentials: 'include',
-            ...init,
-         }
+      const url = `${this.origin}/api/${this.version}/${endpoint}`;
+      const response = await logAsync(
+         `${init?.method || 'GET'}:${truncate(url, 70)}`,
```

Comment:
We can save some space by skipping origin/api/version
```suggestion
         `API ${init?.method || 'GET'}:${truncate(endpoint, 70)}`,
```

Reply:
Yeah, that is better. I was interested in including the hostname... but the endpoint is the crucial one.